### DISCUSSION TITLE
v0.3.2 Beta

### DISCUSSION THREAD

**Discussion by yoshiask** [yoshiask/FluentStore#48] [Announcements]
# What's New
## Features and improvements
- Temporarily moved Fluent Store app data directory to `%ProgramData%` (https://github.com/microsoft/WindowsAppSDK/issues/2788)
- Added support for Winstall's `portable` package type

## Bug fixes
- Resolved an issue that prevented most users from installing Fluent Store (#47)
- Fixed a bug with file system virtualization that caused the app to crash on the splash screen
- Reduced occurrences of download progress bars appearing to move small amounts before jumping to 100%
- Fixed lack of newlines in debug log output

# Known Issues
* Fluent Store profile details cannot be edited
* Signing in with a Microsoft Account does not grant access to private flights
* Resetting settings does not automatically update the UI
* Adding a package to a collection does not show any errors, but does not appear to work
* When some packages are installed, the success message will appear with a stack trace, which should only be included for error messages
* Sometimes the app will crash after successfully launching an installed package

**Full Changelog**: https://github.com/yoshiask/FluentStore/compare/v0.3.1-beta...v0.3.2-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/yoshiask/FluentStore/releases/tag/v0.3.2-beta'>v0.3.2 Beta</a>.</em>

**Comment by OrangeCat2018** [comment]
**Hello,**

I've recently became aware of this project/app, although, upon my attempts to install v0.3.2 (first initial install) I keep receiving:
 _"App installation failed with error message: Class not registered (0x80040154)"_

 
**Wondering if anyone would kindly be able to provide me with some resolution-feedback**

 
 
...
**To go in details about undertaken steps (for troubleshooting purposes):**

Following some quick google searches _using criteria: "Class not registered (0x80040154)"_,
it was suggested that I install the following runtime:
- C++ Runtime v11.0 framework package for Desktop Bridge (Project Centennial)
- C++ Runtime v12 framework package for Desktop Bridge (Project Centennial)

_Although, the installation of those above mentioned-runtime(s) didn't solve this particular issue..._

 

**As for my installation process...**

I've acquired both 
 -   FluentStoreBeta.appinstaller
 -   FluentStoreBeta_0.3.2_2.msixbundle

I first executed **msixbundle** _which resulted to a "Class not registered (0x80040154)" error message._
and then executed **appinstaller** _which also resulted to a "Class not registered (0x80040154)" error message._

 
 
**As for the "executed from" local destination path**

Having altered my OS's default profile basic-folders location..
Both were executed from "D:\Admin\OS\W10\Users\Andre\Download"
(not sure if not being in proper drive letter can be an issue, as I did prior experience a few .exe fail in the past due to this)

**Reply by yoshiask** [reply]
Running the installer from a different location should have no effect on it. Have you by any chance run any debloat scripts on your PC?
